galicaster/mediapackage/repository.py

```diff
@@ -34,7 +34,7 @@ class Repository(object):
 
     def __init__(self, root=None, hostname='',
                  folder_template='gc_{hostname}_{year}-{month}-{day}T{hour}h{minute}m{second}',
-                 logger=None):
+                 logger=None, recoverytype=None):
         """Initializes a repository that will contain a set of mediapackage.
         Args:
             root (str): absolute path to the working folder. ~/Repository used if it is None
```

Comment:
Please, add doc for `recoverytype`.  

The default value must be "full" to keep BC.